src/cloudevent/mocks/helpers.ts

```diff
@@ -27,6 +28,15 @@ export function getBaseCloudEvent<EventType extends CloudEvent<unknown>>(
   } as EventType;
 }
 
+export function extractStringFromStringOrStringParam(
```

Comment:
nit: i think the verb we use internally for "turn an expression into a literal value" is Resolve

Reply:
Updated!